keras_eval/utils.py

```diff
@@ -543,3 +543,20 @@ def results_differential(dataframes, mode='average', round_decimals=4, save_csv_
         differential_dataframe.to_csv(save_csv_path, float_format='%.' + str(round_decimals) + 'f', index=False)
 
     return differential_dataframe
+
+
+def check_result_type(result_csv, individual):
+    '''
+    Checks if the file type is of the required format i.e. individual or average metrics
+    Args:
+        result_csv: csv file name
+        individual: Bool to check if csv file is individual or average metrics
+    Returns: True or False
```

Comment:
I know this is obvious to us, but can we add in the Return description when the function returns `True`, and when `False`? See [example](https://github.com/triagemd/keras-eval/blob/master/keras_eval/utils.py#L169).